config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :users, only: [:new, :create]
+  get '/dashboard', to: 'users#show'
```

Comment:
Can you change this route to '/home' instead of '/dashboard'? I think we were going to use '/dashboard' for the admin.